main.py

```diff
@@ -1,10 +1,51 @@
-import init_django_orm  # noqa: F401
+from json import load
+from os import path
 
-from db.models import Race, Skill, Player, Guild
+import init_django_orm  # noqa: F401
+from db.models import Race, Skill, Guild, Player
 
 
 def main():
-    pass
+    main_path = path.dirname(path.abspath(__file__))
+    with open(main_path + path.sep + "players.json") as f:
```

Comment:
you don't need `main_path`, you can open the file in such a way `with open("players.json")`